practice-three/src/js/script.js

```diff
@@ -212,9 +215,34 @@ const renderUserTable = () => {
 
       updateButtons.forEach((item) => {
         item.addEventListener('click', () => {
-          updateUser(item);
+          getUser(item);
+          submitBtn.className = 'hide-button';
```

Comment:
Should use classList for adding or removing or toggle class:

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList